usmqe_tests/alerting/test_status_alerting.py

```diff
@@ -0,0 +1,350 @@
+
+
+@pytest.mark.testready
+@pytest.mark.author("fbalak@redhat.com")
+@pytest.mark.ansible_playbook_teardown('test_teardown.gluster_volume_stop.yml')
+@pytest.mark.ansible_playbook_setup('test_setup.gluster_volume_stop.yml')
+@pytest.mark.ansible_playbook_setup('test_setup.snmp.yml')
+def test_volume_status_snmp_alert(
+        ansible_playbook,
+        workload_stop_volumes, default_entities):
+    """
+    Check that Tendrl sends correct status alert when volume is stopped.
+    """
+    """
+    :step:
+      Get the messages that arrived in the interval provided by
+      workload_stopped_volume fixture or a little later
+    :result:
+      The list of all relevant messages
+    """
+
+    alerting = Alerting("root")
+    entities = default_entities
+    severity = "WARNING"
+    for volume in workload_stop_volumes["result"]:
+        entities["volume"] = volume
+
+        entities["value"] = "from Started to Stopped"
+        mail_subject, mail_msg = alerting.generate_alert_msg(
+            domain="volume",
+            subject="status",
+            entities=entities)
+        alert_count = alerting.search_snmp(
+            "[{0}], {1}-{2}".format(severity, mail_subject, mail_msg),
+            workload_stop_volumes['start'],
+            workload_stop_volumes['end'])
+        pytest.check(
+            alert_count == 1,
+            "There should be 1 alert:\nSubject: '{0}'\nBody: '{1}'\n"
+            "There is {2}".format(
+                mail_subject, mail_msg, alert_count))
+
+        entities["value"] = "down"
+        mail_subject, mail_msg = alerting.generate_alert_msg(
+            domain="volume",
+            subject="running",
+            entities=entities)
+        alert_count = alerting.search_snmp(
+            "[{0}], {1}-{2}".format(severity, mail_subject, mail_msg),
+            workload_stop_volumes['start'],
+            workload_stop_volumes['end'])
+        pytest.check(
+            alert_count == 1,
+            "There should be 1 alert:\nSubject: '{0}'\nBody: '{1}'\n"
+            "There is {2}".format(
+                mail_subject, mail_msg, alert_count))
+
+    entities["value"] = "unhealthy"
+    mail_subject, mail_msg = alerting.generate_alert_msg(
+        domain="cluster",
+        subject="health",
+        entities=entities)
+    alert_count = alerting.search_snmp(
+        "[{0}], {1}-{2}".format(severity, mail_subject, mail_msg),
+        workload_stop_volumes['start'],
+        workload_stop_volumes['end'])
+    pytest.check(
+        alert_count == 1,
+        "There should be 1 alert:\nSubject: '{0}'\nBody: '{1}'\n"
+        "There is {2}".format(
+            mail_subject, mail_msg, alert_count))
+
+
+@pytest.mark.testready
+@pytest.mark.author("fbalak@redhat.com")
+@pytest.mark.ansible_playbook_teardown('test_teardown.gluster_volume_stop.yml')
+@pytest.mark.ansible_playbook_setup('test_setup.gluster_volume_stop.yml')
+@pytest.mark.ansible_playbook_setup('test_setup.alerts_logger.yml')
```

Comment:
This marker needs to go to the fixture, otherwise the logger would be installed for the run of the test itself, but there would be no data as logger needs to run while the fixture is stressing the system.